release-content/0.14/release-notes/12484_Uniform_point_sampling_methods_for_some_primitive_shapes.md

```diff
@@ -1 +1,56 @@
-TODO
+The ability to perform a variety of random sampling operations has been added to `bevy_math`, gated behind the `rand` feature. These come in a couple of flavors.
+
+First, one can sample random points from the boundaries and interiors of a variety of mathematical primitives:
```

Comment:
It would be nice to spend a sentence listing where this might be useful in the context of game development.